lib/rates/RateProvider.ts

```diff
@@ -12,6 +12,7 @@ class RateProvider {
 
   private cryptoCompare = new CryptoCompare();
 
+  private timer!: NodeJS.Timeout;
```

Comment:
Please add a new line between the declaration of the `timer` variable and the constructor